### DISCUSSION TITLE
How to configure LLC to use via Reverse Proxy

### DISCUSSION THREAD

**Discussion by root9191** [JulianPrieber/littlelink-custom#116] [Questions]
Hello,
could someone tell me how i can use LLC right through the Nginx Proxy Manager?
On the NPM i have the SSL Certs and i wanna control the Logins via Fail2Ban.
But for now i only see the local Adress when i access LLC via Internet and not the public IP. 
Also when i try to access via Internet to my Login Page my Custom Logo is gone.
Is there a Proxy_Pass Environment Variable or do i need to configure the httpd.conf in the Container?

**Comment by JulianPrieber** [comment]
Hi,

I'm not really experienced in doing this. I only ever setup one reverse proxy with LittleLink Custom and the docker version. I mostly only used the NGINX template from the docks and added the proxy settings, and it worked perfectly for me.

I'm not too sure about your solution, tho.

LLC is running on Laravel. You might be able to find something online with that.
From what I could gather, Laravel supports proxy setting by default, but don't quote me on that.

**Essentially:**
```
HTTPS_PROXY={http/https://}PROXY_USER:PROXY_PWD@PROXY:PROXY_PORT
(Combination of Proxy Address:Proxy Password@Proxy Address with port, don’t include http/https)

PROXY=xxxxxx (Proxy Name)
PROXY_PORT=xxxx (Proxy Port)
PROXY_USER=xxxxxxxxx (Proxy User Name)
PROXY_PWD=xxxxxxxxx (Proxy User Password)
PROXY_TYPE=CURLPROXY_HTTP
SSL_VERIFYPEER=true (true if SSL, else false)

Include actual values in HTTPS_Proxy
```
_Again, I'm not sure if this will work, this is just something I dug up while googling the topic 👍_
